paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/manage/ManageSavedPaymentMethodMutatorFactory.kt

```diff
@@ -63,7 +63,6 @@ internal class ManageSavedPaymentMethodMutatorFactory @Inject constructor(
     private fun onPaymentMethodRemoved() {
         val shouldCloseSheet = customerStateHolder.paymentMethods.value.isEmpty()
         if (shouldCloseSheet) {
-            selectionHolder.set(null)
```

Comment:
SavedPaymentMethodMutator does the right thing right before this is called, and has a bunch of unit tests.